doc/source/conduct/code_of_conduct.md

```diff
@@ -121,7 +121,7 @@ You can report issues to the scikit-image Code of Conduct committee, at
 - Egor Panfilov <egor.v.panfilov@gmail.com>
 
 If your report involves any members of the committee, or if they feel they have
-a conflict of interest in handling it, then they will recuse themselves from
+a conflict of interest in handling it, then they will recurse themselves from
```

Comment:
I guess we don't want to apply this "correction."

Reply:
yes, good eye! It looks like I made a mistake in accepting this suggestion. It has been reverted